app/controllers/exhibits_controller.rb

```diff
@@ -1,6 +1,28 @@
 class ExhibitsController < Spotlight::ExhibitsController
   prepend_before_action :find_exhibit
 
+  def create
+    @exhibit.attributes = exhibit_params
+
+    if params[:exhibit].key?("slug") && @exhibit.save
+      @exhibit.roles.create user: current_user, role: 'admin' if current_user
+      ingest_members
```

Comment:
Is the only difference between spotlight's controller and ours this one line? If so, we may be able to decorate this functionality on save, using the find_exhibit stuff we do already.